src/ucp/wireup/select.c

```diff
@@ -1229,8 +1234,10 @@ ucp_wireup_add_bw_lanes(const ucp_wireup_select_params_t *select_params,
             addr_index       = sinfo.addr_index;
             dev_index        = context->tl_rscs[rsc_index].dev_index;
             sinfo.path_index = local_dev_count[dev_index];
+            show_error       = (num_lanes == 0);
```

Comment:
IMO no need this helper var

Reply:
i'd leave it, added this as a documentation instead of comments etc